app/products/calls/components/call_avatar.tsx

```diff
@@ -25,24 +25,31 @@ const getStyleSheet = (props: Props) => {
 
     return StyleSheet.create({
         pictureHalo: {
-            backgroundColor: 'rgba(255,255,255,' + (0.16 * props.volume) + ')',
+            backgroundColor: 'rgba(61, 184, 135,' + (0.24 * props.volume) + ')',
             height: baseSize + 16,
             width: baseSize + 16,
             padding: 4,
             marginRight: 4,
             borderRadius: (baseSize + 16) / 2,
         },
         pictureHalo2: {
-            backgroundColor: 'rgba(255,255,255,' + (0.24 * props.volume) + ')',
+            backgroundColor: 'rgba(61, 184, 135,' + (0.32 * props.volume) + ')',
             height: baseSize + 8,
             width: baseSize + 8,
-            padding: 4,
+            padding: 3,
             borderRadius: (baseSize + 8) / 2,
         },
         picture: {
             borderRadius: baseSize / 2,
             height: baseSize,
             width: baseSize,
+            marginBottom: 5,
+        },
+        voiceShadow: {
+            shadowColor: 'rgb(61, 184, 135)',
+            shadowOffset: {width: 0, height: 0},
+            shadowOpacity: 1,
+            shadowRadius: 10,
```

Comment:
> and closer (but not exact) for android

I believe there is a library that you could try and see if it works for you here https://github.com/SrBrahma/react-native-shadow-2

Reply:
@enahum I also found that one, but decided against it because I didn't want to add another dependency, and the android shadow is "close enough." But open to adding it, if you think it's worth it?

Reply:
Up to you and the UX team, for me the "close enough" looks kinda bad 🤣

Reply:
Hah, good point. @abhijit-singh your call! :)